### PR TITLE
PAYARA-2427 Payara Micro Deployed Application Produces Log Spam

### PR DESCRIPTION
**Pull request by MattGill98**
Old descriptor:
~~~
[2018-02-08T16:49:26.982+0000] [] [INFO] [] [PayaraMicro] [tid: _ThreadID=1 _ThreadName=main] [timeMillis: 1518108566982] [levelValue: 800] [[
  
Instance Configuration
Host: Matt-Desktop-Arch.localdomain
HTTP Port(s): 8080 
HTTPS Port(s): 
Instance Name: payara-micro
Instance Group: no-cluster
Deployed: jaxrs-hello-world-1.0-SNAPSHOT ( jaxrs-hello-world-1.0-SNAPSHOT war /jaxrs-hello-world-1.0-SNAPSHOT [ < jsp *.jspx >< uk.me.mattgill.samples.jaxrs.hello.world.service.JaxrsApplication /* >< jsp *.jsp >< default / > ] )
Deployed: config-api-sample-1.0-SNAPSHOT ( config-api-sample-1.0-SNAPSHOT war /config-api-sample-1.0-SNAPSHOT [ < jsp *.jspx >< uk.me.mattgill.samples.config.api.test.JaxrsApplication /* >< jsp *.jsp >< default / > ] )

]]
~~~
New descriptor:
~~~
[2018-02-12T09:22:13.147+0000] [] [INFO] [] [PayaraMicro] [tid: _ThreadID=1 _ThreadName=main] [timeMillis: 1518427333147] [levelValue: 800] [[
  
{
    "Instance Configuration": {
        "Host": "Matt-Desktop-Arch.localdomain",
        "Http Port(s)": "8081",
        "Https Port(s)": "8182",
        "Instance Name": "payara-micro",
        "Instance Group": "no-cluster",
        "Deployed": [
            {
                "Name": "jaxrs-hello-world-1.0-SNAPSHOT",
                "Type": "war",
                "Context Root": "/jaxrs-hello-world-1.0-SNAPSHOT"
            }
        ]
    }
}]]
~~~
New descriptor with `--showservletmappings`:
~~~
[2018-02-12T09:22:38.174+0000] [] [INFO] [] [PayaraMicro] [tid: _ThreadID=1 _ThreadName=main] [timeMillis: 1518427358174] [levelValue: 800] [[
  
{
    "Instance Configuration": {
        "Host": "Matt-Desktop-Arch.localdomain",
        "Http Port(s)": "8081",
        "Https Port(s)": "8182",
        "Instance Name": "payara-micro",
        "Instance Group": "no-cluster",
        "Deployed": [
            {
                "Name": "jaxrs-hello-world-1.0-SNAPSHOT",
                "Type": "war",
                "Context Root": "/jaxrs-hello-world-1.0-SNAPSHOT",
                "Mappings": {
                    "*.jspx": "jsp",
                    "/*": "uk.me.mattgill.samples.jaxrs.hello.world.service.JaxrsApplication",
                    "*.jsp": "jsp",
                    "/": "default"
                }
            }
        ]
    }
}]]
~~~

### CODE REVIEW

File: appserver/extras/payara-micro/payara-micro-boot/pom.xml [hunk 1]
```diff
@@ -49,8 +49,9 @@
     </parent>
     <groupId>fish.payara.micro</groupId>
     <artifactId>payara-micro-boot</artifactId>
-    <description>Payara Micro Boot</description>
     <packaging>glassfish-jar</packaging>
+    <name>Payara Micro Boot</name>
```
Comment (Cousjava) [inline appserver/extras/payara-micro/payara-micro-boot/pom.xml]: Update copyright year

File: appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/data/InstanceDescriptor.java [hunk 2]
```diff
@@ -57,72 +57,72 @@
      * @param obj
      * @return
      */
-    public boolean equals(Object obj);
+    boolean equals(Object obj);
```
Comment (Cousjava) [inline appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/data/InstanceDescriptor.java]: Update copyright year again

File: appserver/extras/payara-micro/payara-micro-core/pom.xml [hunk 3]
```diff
@@ -9,8 +9,8 @@
     <groupId>fish.payara.micro</groupId>
     <artifactId>payara-micro-core</artifactId>
     <packaging>glassfish-jar</packaging>
-    <name>Payara Micro Edition Core</name>
-    <description>Payara Micro Edition Modules</description>
+    <name>Payara Micro Core</name>
```
Comment (Cousjava) [inline appserver/extras/payara-micro/payara-micro-core/pom.xml]: Needs copyright notice adding to file

File: appserver/payara-appserver-modules/payara-micro-service/src/main/java/fish/payara/appserver/micro/services/data/InstanceDescriptorImpl.java [hunk 4]
```diff
@@ -17,11 +17,13 @@ and Distribution License("CDDL") (collectively, the "License").  You
  */
 package fish.payara.appserver.micro.services.data;
 
+import java.io.StringWriter;
```
Comment (Cousjava) [inline appserver/payara-appserver-modules/payara-micro-service/src/main/java/fish/payara/appserver/micro/services/data/InstanceDescriptorImpl.java]: Needs updated copyright notice

File: appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/data/InstanceDescriptor.java (hunk 2 above)
Comment (smillidge) [reply appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/data/InstanceDescriptor.java]: Why are these no longer public? I believe these methods are used by other packages.

Comment (MattGill98) [reply appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/data/InstanceDescriptor.java]: Might be wrong, but I vaguely remembered @arjantijms pointing out in a previous PR of mine that the methods in interfaces are public by default

Comment (arjantijms) [reply appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/data/InstanceDescriptor.java]: @MattGill98 @smillidge The type is an interface indeed:

```java
public interface InstanceDescriptor
```

Interface methods are public by default and the public access modifier is redundant there.

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

**Comment by arjantijms** [comment]
jenkins quick test payara 5 please

**Pull request merged**